### PR TITLE
Sangeetparashar/yld 25 stake page btc only

### PR DESCRIPTION
**Pull request by sangeetparashar**
* Revamp of stake page (mobile and desktop version)
* Added data provider to fetch prices

### CODE REVIEW

File: src/app/components/Transactions/Transactions.tsx [hunk 1]
```diff
@@ -312,7 +312,7 @@ const Transactions: React.FC<{
         nextBlockParams: getCurrentGlobalParamsVersion(height + 1, versions),
       };
     },
-    refetchInterval: 60000, // 1 minute
+    refetchInterval: 300000, // 1 minute
```
Comment (MattNguyen) [inline src/app/components/Transactions/Transactions.tsx]: why change this to 5 minutes?

File: src/app/components/Transactions/Transactions.tsx [hunk 2]
```diff
@@ -443,23 +443,31 @@ const Transactions: React.FC<{
...
+                    }
+                    onWithdraw={() =>
+                      handleModal(delegation.stakingTxHashHex, MODE_WITHDRAW)
+                    }
+                    intermediateState={intermediateDelegation?.state}
+                    globalParamsVersion={
+                      paramWithContext?.nextBlockParams?.currentVersion
+                    }
+                  />
+                  <Table.Row
```
Comment (MattNguyen) [inline src/app/components/Transactions/Transactions.tsx:512]: How is this change related to the stake page?

File: src/app/context/AssetContext.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+// context/AssetsContext.tsx
```
Comment (MattNguyen) [inline src/app/context/AssetContext.tsx]: Remove

File: src/app/context/DataContext.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+// context/DataContext.tsx
```
Comment (MattNguyen) [inline src/app/context/DataContext.tsx]: remove

File: src/app/stake/page.tsx [hunk 5]
```diff
@@ -5,103 +5,158 @@ import Image from "next/image";
...
+                        </span>
+                      </span>
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 items-center h-full ">
+                    <div className="text-yieldi-brown text-xl font-normal ">
+                      $0.00
+                    </div>
+                    <div className="text-yieldi-brown-light text-sm font-normal">
+                      0.0 {asset.assetSymbol}
```
Comment (MattNguyen) [inline src/app/stake/page.tsx]: do not hardcode

File: src/app/stake/page.tsx [hunk 6]
```diff
@@ -5,103 +5,158 @@ import Image from "next/image";
...
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 ">
+                    <div className="text-yieldi-brown text-xl font-normal flex items-center h-full  ">
+                      0.00 {asset.assetSymbol}
```
Comment (MattNguyen) [inline src/app/stake/page.tsx]: do not hardcode

File: src/app/config/StakedAssets.tsx [hunk 7]
```diff
@@ -14,6 +15,7 @@ export const assets: StakeAsset[] = [
   {
     assetName: "Native ETH",
```
Comment (MattNguyen) [inline src/app/config/StakedAssets.tsx:16]: Should remove eth and eth lsts until we actually integrate with eigen.

cc @kiasaki

**Review by MattNguyen — CHANGES_REQUESTED** [review]
also left comments in vercel. main thing is to make sure we are using real values.

File: src/app/components/Transactions/Transactions.tsx (hunk 2 above)
Comment (sangeetparashar) [reply src/app/components/Transactions/Transactions.tsx:512]: This doesnt change the stake page. I noticed spacing was incorrect in our details page and applied the fix we're using in finality provides, and all following tables

File: src/app/config/StakedAssets.tsx (hunk 7 above)
Comment (sangeetparashar) [reply src/app/config/StakedAssets.tsx:16]: In the process of moving to .env files and will disable until we're ready for it.

File: src/app/components/Transactions/Transactions.tsx (hunk 1 above)
Comment (sangeetparashar) [reply src/app/components/Transactions/Transactions.tsx:315]: initally changed it to 5 minutes for testing, and was being throttled. Changed it back to 1 minute

Comment (sangeetparashar) [reply src/app/components/Transactions/Transactions.tsx]: did not mean to make this change

Comment (sangeetparashar) [reply src/app/components/Transactions/Transactions.tsx]: reverted

File: src/app/stake/page.tsx [hunk 8]
```diff
@@ -5,103 +5,177 @@ import Image from "next/image";
...
+                        <br />
+                        <span className="text-yieldi-brown-light font-gt-america text-sm font-normal">
+                          {asset.assetName}
+                        </span>
+                      </span>
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 items-center h-full ">
+                    <div className="text-yieldi-brown text-xl font-normal ">
+                      $0.00
```
Comment (MattNguyen) [inline src/app/stake/page.tsx]: we need to get the real tvl here

File: src/app/stake/page.tsx [hunk 9]
```diff
@@ -5,103 +5,177 @@ import Image from "next/image";
...
+                    </div>
+                    <div className="text-yieldi-brown-light text-sm font-normal">
+                      {isConnected
+                        ? (
+                            satoshiToBtc(btcWalletBalanceSat) * asset.price
```
Comment (MattNguyen) [inline src/app/stake/page.tsx]: should be tvl * asset.price

File: src/app/stake/page.tsx [hunk 10]
```diff
@@ -5,103 +5,177 @@ import Image from "next/image";
...
+                          ).toString()
+                        : "-"}
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 ">
+                    <div className="text-yieldi-brown text-xl font-normal flex items-center h-full  ">
+                      {isConnected
+                        ? (
+                            satoshiToBtc(btcWalletBalanceSat) * asset.price
```
Comment (MattNguyen) [inline src/app/stake/page.tsx]: Remove `* asset.price`. Just need the btc wallet balance

Comment (sangeetparashar) [reply src/app/stake/page.tsx]: done
